common/changes/@microsoft/rush/reconfigure-watch_2021-02-13-00-01.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "Replace \"--watch\" with a \"watchForChanges: true\" setting in command-line.json, since running in watch mode is not automatically compatible with downstream file watchers, and use of such is the expected common scenario.",
+      "type": "none"
```

Comment:
Given that it's "experimental", we could probably do this as a SemVer PATCH bump